h/schemas/annotation.py

```diff
@@ -311,3 +313,99 @@ def _target_selectors(targets):
         return targets[0]['selector']
     else:
         return []
+
+
+class SearchParamsSchema(colander.Schema):
+    _separate_replies = colander.SchemaNode(
+        colander.Boolean(),
+        missing=False,
+        description="Return a seperate set of annotations and their replies.",
+    )
+    sort = colander.SchemaNode(
+        colander.String(),
+        validator=colander.OneOf(["created", "updated", "group", "id", "user"]),
+        missing="updated",
+        description="The field by which annotations should be sorted.",
+    )
+    limit = colander.SchemaNode(
+        colander.Integer(),
+        validator=colander.Range(min=0, max=LIMIT_MAX),
```

Comment:
It is possible there are existing API users out there passing in illegal values for limit or offset, so even though we are not making a change to the limits, there is a small risk that it causes a problem. I think we should still merge this, but we should keep an eye out and be prepared to adjust quickly if necessary.

Reply:
I feel safe about using a limit of 10000 cause right now that is our limit anyway. I feel less safe about decreasing it to 2000 without first seeing what limits are being used today. I think I'll leave it at 10000 for now and add an event in to record the value and then based on that info we can decrease it to a more reasonable number w/o breaking anyone.